monai/transforms/io/dictionary.py

```diff
@@ -59,6 +59,7 @@ def __init__(
         dtype: DtypeLike = np.float32,
         meta_key_postfix: str = "meta_dict",
         overwriting: bool = False,
+        image_only: bool = False,
```

Comment:
docstring updated

Reply:
> perhaps also revise the docstring for this new option, it might be confusing... could you also help confirm this change @Nic-Ma

Looks good to me.
Thanks.